backend/app/views/spree/admin/orders/edit.html.erb

```diff
@@ -18,6 +18,8 @@
   <%= render 'spree/admin/orders/risk_analysis', latest_payment: @order.payments.reorder("created_at DESC").first %>
 <% end %>
 
+<div data-hook="add_product_button" />
```

Comment:
I'd try to use a more generic name since other extensions could use this hook in other ways.

Reply:
@kennyadsl makes sense 👍
I changed the name.

Reply:
@kennyadsl makes sense 👍
I changed the name to `admin_order_edit_sub_header`